configs/infracost.json

```
@@ -19,8 +30,8 @@
     "lvl2": "article h2",
     "lvl3": "article h3",
     "lvl4": "article h4",
-    "lvl5": "article h5, article td:first-child",
-    "text": "article p, article li, article td:last-child"
+    "lvl5": "article h5, article",
+    "text": "article p, article li, article"
```

```suggestion
    "lvl5": "article h5",
    "text": "article p, article li, article td"
```

I assume this is what you'd achieve

---

I think so, we have `td` that have `code` too, e.g.:
```
<tr><td>Bastion</td><td><code>azurerm_bastion_host</code></td><td></td></tr>
```

Should I include that too? `article p, article li, article td, article td code`

---

It should be enough like that, but feel free to re-open a PR if not

---

Thanks!